psycop/common/model_training_v2/metrics/binary_metrics/binary_auroc.py

```diff
@@ -5,12 +5,9 @@
 from psycop.common.model_training_v2.metrics.binary_metrics.base_binary_metric import (
     BinaryMetric,
 )
-from psycop.common.model_training_v2.presplit_preprocessing.polars_frame import (
-    PolarsFrame,
-)
 
 
 class BinaryAUROC(BinaryMetric):
-    def calculate(self, y_true: PolarsFrame, y_pred: pl.Series) -> CalculatedMetric:
+    def calculate(self, y_true: pl.Series, y_pred: pl.Series) -> CalculatedMetric:
         # sklearn expects MatrixLike | ArrayLike which PolarsFrame and pl.Series fulfill but are not typed as
```

Comment:
We talked about this a bit before. The type:ignore makes me worried, perhaps metrics should just support pd.Series instead?